CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
-# Contributing to TIDES 
-Thank you for contributing to the TIDES Project.  This document defines the roles and process for 
+# Contributing to TIDES
+Thank you for contributing to the TIDES Project.  This document defines the roles and process for
 contributing to the project and documents the governance roles and approach for decision-making.
 
 ## Contents
```

Comment:
pre-commit is telling me that lists need to be surrounded by blank lines. Did you get that warning?

Reply:
ditto